src/main/webapp/dashboard.js

```diff
@@ -93,6 +93,11 @@ function createAboutMe(user) {
   userBioElement.innerText = user.bio;
 
   userGithubButton.href = user.gitHubURL;
+  if (user.email == null) {
+    userEmailButton.style.display = 'none';
+  }
+  userEmailButton.href = 'mailto:' + user.email;
+  // TODO: only send email when authorized
```

Comment:
I believe this is resolved with the changes above that add an authorization check to the `UserData` constructor?

Reply:
yeah, good catch!